changes/test_tasks.py

```diff
@@ -1030,7 +1032,9 @@ def assert_openhim_request(request):
             registrant_id="identity-uuid", data={"faccode": "123456"}, source=source
         )
 
-        process_engage_helpdesk_outbound.delay("27820001001", message_id).get()
+        process_engage_helpdesk_outbound.apply_async(
+            args=["27820001001", message_id], task_id=message_id
```

Comment:
This shouldn't be the message ID, this should be a UUID. We want the message ID and the task UUID to be two different things.